### DISCUSSION TITLE
Possible new types for Al-Anon

### DISCUSSION THREAD

**Discussion by eflashlogistics** [code4recovery/12-step-meeting-list#736] [Ideas]
### Contact Details

lucas.3@aol.com

### Requested Feature/Enhancment

Hi! I am using your plugin for al-anon and I saw that you said to suggest other meeting types. Some I will add to php:

1. Young Adults (WSO approved-used on their site under advanced)
2. LGBTQIA+ (same)

One I use for my district and will add 
3. Literature meeting

### TSML Version

Latest (default)

### Wordpress Version

Latest (default)

**Comment by tech2serve** [comment]
@code4recovery/tsml-maintainers, The two types the OP proposed are not included in the Al-Anon defaults, but they are probably reasonable. Do we want to add them as defaults?

@eflashlogistics, in the meantime, you can add them in your custom `functions.php`. Please read our FAQ for how to do that.

**Reply by brianw-area24** [reply]
With WSO using the first 2, it makes sense for the plugin to include them

**Reply by tech2serve** [reply]
@eflashlogistics, replies to this comment (vice a new comment) will make following the discussion a bit easier.

We can create a developer issue for the two new types after a bit more discussion here. 

"Young Adults" is straight forward on the surface, and we could use "YA" for the code. I'm curious though, when I look through our code we currently have "Y" for "Alateen", but Al-Anon's own meeting list finder doesn't have that. So I guess I'm trying to understand the difference between "Young Adults" and Alateen?

Regarding "LGBTQIA+" as a type, this seems easy to add, but TSML currently supports L, G and T as separate types for Al-Anon. Inside of AA there have been some discussions of collapsing these types into one. I personally support this, and I believe L, G and T should be migrated to LGBTQIA+. This would match the Al-Anon website, but it might not align with some Al-Anon groups around the country. Do you have a sense for how this might be handled? Has WSO provided any guidance?

**Reply by eflashlogistics** [reply]
I see that now! I honestly was replying via email before. 

Alateen would be under 18. Young adults is a pretty loose term but it is 18+. Alateen isn't listed online, or at least publicly. Due to the nature of it being minors and liability. 

So yeah that advanced search section was a recent addition and much debate was had to create it. I reached out to a friend with that question and she had a lot of feedback. Her opinion was in addition to having LGBTQIA+ in addition to singular, and a lot around gender identity vs sexual preference. She sent a list of meetings and they all had it in the name and varied greatly. Gender non-conforming seemed to be a good blanket statement. 

WSO would probably want to follow their advanced search section.

**Comment by eflashlogistics** [comment]
Will do. Wso is using the first two. Our district uses the third, literature study. 

Ill checkin when we finish our listings. I also work with our world service office a lot and am really hoping they can implement this along with our app. 

We are currently establishing a new electronic area which I think you all call region, and need a whole new format. 

Yours is really great. 

Thanks.

**Comment by eflashlogistics** [comment]
If you go to electronic meeting search online and hit advanced in the search options it shows the approved categories they use. 

https://al-anon.org/al-anon-meetings/electronic-meetings/

**Comment by VickiB11** [comment]
WSO has updated the Group Records Change Form to include a Participants section.  Types not currently included on the TSML are: Young Adults, LGBTQIA+, Parents of Alcoholics, People of Color.  Here’s a link to the GR1 form: https://al-anon.org/pdf/Group-Records-Change-EN.pdf
It would be appropriate for the TSML to add these Participants types to the Al-Anon defaults.

**Comment by joshreisner** [comment]
Hi @VickiB11 @eflashlogistics apologies for the long hiatus here. 

Is this still of interest? The request was to add:

* Young Adults
* LGBTQIA+
* Parents of Alcoholics
* People of Color

**Comment by VickiB11** [comment]
Yes, we are still using those types. VickiOn Dec 8, 2022, at 10:45 AM, Josh Reisner ***@***.***> wrote:﻿
Hi @VickiB11 @eflashlogistics apologies for the long hiatus here.
Is this still of interest? The request was to add:

Young Adults
LGBTQIA+
Parents of Alcoholics
People of Color

**Comment by eflashlogistics** [comment]
I would agreeI manually added those types to our district site Sent from my iPhoneOn Dec 8, 2022, at 7:45 AM, Josh Reisner ***@***.***> wrote:﻿
Hi @VickiB11 @eflashlogistics apologies for the long hiatus here.
Is this still of interest? The request was to add:

Young Adults
LGBTQIA+
Parents of Alcoholics
People of Color

**Comment by joshreisner** [comment]
Ok great. We'll include them in the next release. What codes did you use when you added them locally? I want to use the same ones so that it doesn't create duplicate types for you. Thanks

**Comment by eflashlogistics** [comment]
I can match to the ones Vicki sent. So it matches WSO. I only represent one district in california so I would imagine that would fit better globally. Thank you. So cool to be here. Sent from my iPhoneOn Dec 8, 2022, at 10:25 AM, Josh Reisner ***@***.***> wrote:﻿
Ok great. We'll include them in the next release. What codes did you use when you added them locally? I want to use the same ones so that it doesn't create duplicate types for you. Thanks
